index.js

```diff
@@ -1 +1,37 @@
 // implement your API here
+const express = require("express");
+const db = require("./data/db.js");
+
+const server = express();
+
+server.use(express.json());
+
+server.post("/api/users", (req, res) => {
+  const newUser = req.body;
+  console.log("newUser", newUser);
+  db.user
```

Comment:
So your code here will not work because you're attempting to call your `add()` function from `db.user`, but take a look at your exports in your db.js file:

`module.exports = {
  find,
  findById,
  insert,
  update,
  remove,
};`

For `db.user` to be a valid way to access your helper functions, your db.js file would contain this instead:
`module.exports = {
 user = {
  find,
  findById,
  insert,
  update,
  remove,
 }
};`
In short, you can remove the `.user` from your db call